pyproject.toml

```diff
@@ -66,6 +66,9 @@ report-polarion = [
     "tmt[report-junit]",
     "tmt[export-polarion]",
     ]
+link-jira = [
+    "jira>=3.5.0",
```

Comment:
@martinhoyer, shouldn't this be `jira>=3.5, <4`? Otherwise we fail on `el9`:

```
  - nothing provides (python3.9dist(jira) < 4 with python3.9dist(jira) > 3.5) needed by tmt+link-jira-1.37.0.dev44+gf52d782a.d20240925-%autorelease.noarch from @commandline
```

Reply:
Yes yes, thanks. "typo". I even mentioned it above.

Reply:
Ok. And just to clarify, why the upper limit `<4` is needed?

Reply:
not needed, it's just that major version bumps means breaking changes (at least supposedly). Since EPEL9 and upstream are on the same major version, I'm thinking it's a good practice.  
fwiw, I've suggested bumping the version in pre-commit config from 3.2, only then realizing it's already 3.5 in pyproject :)

Reply:
Fixed in 5d2d27d2.